README.md

```diff
@@ -296,6 +298,71 @@ and then change its colors, the items missed would fallback to use the default c
 Please also notice that an empty theme is **NOT** supported due to
 [a bug in serde lib](https://github.com/dtolnay/serde-yaml/issues/86).
 
+### Icon Theme
+
+Icon theme can be configured in a fixed location, `LSD_CONFIG_DIR/themes/icons.yaml`,
```

Comment:
I think we can mention `$XDG_CONFIG_DIR/lsd` here instead of `LSD_CONFIG_DIR` similar to how it is in colors.